datalake/backend/tests/collection.json

```diff
@@ -0,0 +1,104 @@
+{
+    "id": "test_dataset",
+    "stac_version": "1.0.0-beta.2",
+    "description": "This is dataset description.",
+    "links": [
+        {
+            "rel": "item",
+            "href": "s3://s3-bucket-name/datasets/xyz/1234.json",
```

Comment:
We'll be traversing these URLs as well, so they should also be `S3Objects`.

Reply:
I used minimal STAC metadata file content now.